vllm/platforms/cuda.py

```diff
@@ -167,13 +170,13 @@ def check_and_update_config(cls, vllm_config: "VllmConfig") -> None:
 
             from vllm.attention.ops.flashmla import is_flashmla_supported
             if use_flashmla and is_flashmla_supported()[0] \
-                and cache_config.block_size != 64:
-                cache_config.block_size = 64
+                and logic_block_size != 64:
+                logic_block_size = 64
```

Comment:
Need to handle block_size 64 here https://github.com/vllm-project/vllm/blob/e40827280b225bf0a0797cc9842fc3cdfea8ebdf/vllm/model_executor/models/config.py#L376-L392